From fd0d453c3d340790719ba3e97a51d0456722a234 Mon Sep 17 00:00:00 2001
From: Matt Turner <mattst88@gmail.com>
Date: Fri, 7 Aug 2026 12:18:18 +0100
Subject: [PATCH] configure: Fix -WCClinker typo; should be -XCClinker

Fixes: 5f46ba49 ("Made -nostdlib depend on exception support")

Backport of 8e6a348 to v1.8.3 by Tom Pollard <tom.pollard@codethink.co.uk>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 14762b84..bbe8bb9c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -306,7 +306,7 @@ AM_COND_IF([SUPPORT_CXX_EXCEPTIONS],
             AC_SEARCH_LIBS([_Unwind_Resume], [gcc_s gcc],
                            [AS_IF([test "$ac_cv_search__Unwind_Resume" != "none required"],
                                   [AC_SUBST([UNW_CRT_LIBADD],  ["-lc $ac_cv_search__Unwind_Resume"])
-                                   AC_SUBST([UNW_CRT_LDFLAGS], ["-WCClinker -nostdlib"])])]
+                                   AC_SUBST([UNW_CRT_LDFLAGS], ["-XCClinker -nostdlib"])])]
                            [-lc])
             LDFLAGS="$save_LDFLAGS"]
 )
-- 
2.53.0

